PersonaStats.Rmd

```diff
@@ -0,0 +1,287 @@
+---
+title: "Personas"
+author: "Kate Isaac"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r}
+library(tidyverse)
+library(dplyr)
+library(data.table)
+library(magrittr)
+library(here)
+```
+
+```{r}
+resultsTidy <- readRDS(here("data/resultsTidy.rds"))
```

Comment:
Is the intention to only run this locally?

Reply:
More than happy to set it up so that it uses the knit_child process of the others! I only set it up locally while I was working on it. 

How should the output be setup if we change it to render/work with the website?